README.md

```diff
@@ -0,0 +1,53 @@
+# Minion 3
+Flatpak wrapper for Minion, a premier addon manager.
+
+## Disclaimer (important to read)
+
+### Regarding Hardware Acceleration
+
+As hardware acceleration doesn't work well with XWayland under Wayland with the closed-source Nvidia drivers, hardware acceleration was disabled to have higher coverage for GPUs.
+
+One may enable hardware acceleration by running `flatpak override --user --device=dri gg.minion.Minion`. Be wary, this will cause flickering for Minion if you're using the proprietary Nvidia driver while running Wayland.
+
+### First start
```

Comment:
just to clarify expectations, no user will read this.

Reply:
I know, but just put it there just in case so that potentially I don't get issue reports and PRs in the future for dri being enabled.